pennylane/interfaces/tensorflow_autograph.py

```diff
@@ -33,6 +34,16 @@
 )
 
 
+def _flatten(x):

```

Comment:
How about `_flatten_list` or `_flatten_nested_list`? The name is quite general as of now.